tutorials/mongoose/deleteMany.md

```diff
@@ -0,0 +1,41 @@
+If you wish to delete several documents in mongoose, use the `deleteMany()` function.
+It takes up to three parameters:
+- **condition**, what a document should contain to be eligible for deletion.
+You can omit this property to delete all documents in the model.
+- **options**, how to modify the deletion process.
+- **A callback function(err,res)**, use this if you are not using `async/await`.
```

Comment:
Omit the callback argument, not necessary since [callbacks](https://masteringjs.io/tutorials/fundamentals/callbacks) are rarely used anymore.